inst/odin/carehomes.R

```diff
@@ -454,8 +455,8 @@ I_with_diff_trans[, ] <-
 ## NOTE: "age groups" 1-17 are age groups, 18 are CHW and 19 CHR. Here we apply
 ## beta to all contacts *except* within care home contacts
 s_ij[, ] <- m[i, j] * sum(I_with_diff_trans[j, ])
-s_ij[1:17, 1:19] <- beta * s_ij[i, j]
-s_ij[18:19, 1:17] <- beta * s_ij[i, j]
+s_ij[1:n_age_groups, 1:n_groups] <- beta * s_ij[i, j]
+s_ij[(n_age_groups + 1):n_groups, 1:n_age_groups] <- beta * s_ij[i, j]
```

Comment:
Perhaps the comment about this section needs editing now?

Reply:
I thought that comment was still reasonable